src/SpecialFunctions.jl

```diff
@@ -71,10 +71,20 @@ end
 export sinint,
        cosint
 
+export lambertw, lambertwbp
+
+const omega_const_bf_ = Ref{BigFloat}()
+function __init__()
+    # allocate storage for this BigFloat constant each time this module is loaded
+    omega_const_bf_[] =
+        parse(BigFloat,"0.5671432904097838729999686622103555497538157871865125081351310792230457930866845666932194")
```

Comment:
Isn't `big"0.56..."` better than `parse(...)`?

Reply:
If you check, for instance with `@less big"1.0"`, you see that the macro does more work.

Reply:
You're right. `@code_llvm big"0.56..."` gives a LOT more code.

Reply:
Also, `big"..."` doesn't respect current set precision:
```julia

Reply:
Is that intended behavior of `big"..."`? Or should one raise an issue?